paper/paper.md

```diff
@@ -0,0 +1,109 @@
+---
+title: 'MNE-BIDS'
+tags:
+  - Python
+  - BIDS
+  - MNE
+  - neuroimaging
+  - MEG
+  - EEG
+authors:
+  - affiliation: 1
+    name: Stefan Appelhoff
+    orcid: 0000-0001-8002-0877
+  - affiliation: 2
+    name: Mainak Jas
+    orcid: 0000-0002-3199-9027    
+  - affiliation: 3
+    name: Chris Holdgraf
+    orcid: 0000-0002-2391-0678
+  - affiliation:
+    name:
+    orcid:
+affiliations:
+- index: 1
+  name: Center for Adaptive Rationality, Max Planck Institute for Human Development, Berlin, Germany
+- index: 2
+  name: Télécom ParisTech, France  
+- index: 3
+  name: University of California at Berkeley
+- index:
+  name:
+date: 25 October 2019
+bibliography: paper.bib
+---
+
+# Summary
+
+With the emergence of the Brain Imaging Data Structure
+(``BIDS``; [@Gorgolewski2016]) in 2016, the scientific community received a
+standard to organize and share data in the broad domain of neuroscience.
+
+Originally limited to MRI data types, BIDS is continuously being extended and
+by now also supports other neuroimaging modalities such as MEG [@Niso2018] and
+EEG [@Pernet2019].
+
+BIDS prescribes how complex experiment data should be structured and which
+metadata should be encoded next to the raw data. This set of guidelines
+allows users to reap several benefits, such as:
+
+1. **sharing data within a lab and between labs:** Through reading the BIDS
+   specification, users are empowered to understand any BIDS dataset
+   without requiring specialist knowledge.
+1. **data validation:** Dedicated software can check whether the rules of
+   BIDS are being followed in a dataset, and alert users to missing or broken
+   data (see
+   [bids-validator](https://github.com/bids-standard/bids-validator)).
```

Comment:
I think there should be some statement about the fact that testing is done with the validator so that we guarantee the outputs are BIDS compliant

Reply:
Thanks! I added a sentence for this.